src/deploy-cromwell-on-azure/deploy-cromwell-on-azure.csproj

```diff
@@ -72,6 +73,7 @@
     <EmbeddedResource Include="scripts\mount-data-disk.sh" />
     <EmbeddedResource Include="scripts\mount.blobfuse" />
     <EmbeddedResource Include="scripts\mount_containers.sh" />
+    <EmbeddedResource Include="scripts\mysql\init-user-postgre.sql" />
```

Comment:
Unless it requires significant re-work (to handle existing mysql directory), it would make sense to rename this directory from "mysql" to something else, OR, to have a "postgresql" directory for this file

Reply:
I agree it's less than ideal but can we save that re-naming for when we do deprecate docker mysql? I can make a note on the issue, i just don't want to open up for any possible errors with path names.

Reply:
This file can go away - see comment above.

Reply:
deleted the file